packages/unified-lint-rule/lib/index.js

```diff
@@ -39,7 +39,7 @@ export function lintRule(meta, rule) {
   return plugin
 
   /** @type {import('unified').Plugin<[unknown]|Array<void>>} */
-  function plugin(raw) {
+  function plugin(raw, fileSet) {
```

Comment:
```suggestion
  function plugin(raw) {
```

If we don’t need `fileSet`, then we shouldn’t pass it.

Reply:
- [x] Applied.